crates/db_views_actor/src/community_view.rs

```diff
@@ -232,6 +232,13 @@ impl<'a> CommunityQueryBuilder<'a> {
       query = query.filter(community_block::person_id.is_null());
     }
 
+    //Hide if hidden and not subscribed
+    query = query.filter(
+      community::hidden
+        .eq(false)
+        .or(community_follower::person_id.eq(person_id_join)),
+    );
```

Comment:
> I thought the only point was to hide its posts and comments from All.

Late me take a deeper look and get back to you on that.

Reply:
I see, I wanted to keep stuff off the trending section but I can see how that might not be ideal for everyone's usage. Especially if you want to moderate hidden communities and want to go see what is hidden. I'll remove this code. Thank you for pointing that out. It would be nice to be able to hide certain communities from trending.

Reply:
Actually, I'm gonna remove it from hot sort. I'll tag you on that section of the code next. I think that will work out really well.